templates/system/page.tpl.php

```diff
@@ -210,6 +211,23 @@
 
   <div class="page-content">
     <div class="container-fluid">
+
+    <?php if ($settings['improved'] && $settings['improved_header'] == 'basic' && (!empty($page['custom_title']) || !empty($title))): ?>
```

Comment:
Please change
`$settings['improved_header'] == 'basic'`
to
`$settings['improved_header'] === 'basic'`